syntax = "proto3";

package m4u_interfaces;

option java_package = "com.m4u_interfaces.response";
option java_outer_classname = "ResponseProtos";

message Response {
   int32  id            = 1;
   string msg           = 2;   
}

message ResponseBlock {
  repeated Response response = 1;
}
